src/repr.ttl

```diff
@@ -55,19 +61,54 @@ repr:Coordinates rdf:type owl:Class ;
                                  ] .
 
 
+###  http://sweetontology.net/repr/1D
+repr:1D rdf:type owl:Class ;
+       rdfs:subClassOf repr:Representation ,
```

Comment:
I would have to review the whole SWEET approach in general, but at least in this case, it seems to me that `repr:1D`, `repr:2D` and `repr:3D` should be **instances** and not **classes**  themselves: you wouldn't be creating instances of `repr:1D` for example. 

Perhaps something like this would be more appropriate?

```
repr:1D rdf:type repr:Representation ,
                       [ rdf:type owl:Restriction ;
                         owl:onProperty mrela:hasDimension ;
                         owl:hasValue 1
                       ] .
```

Reply:
First a pragmatic question: would this affect any existing infrastructure that uses sweet and hardcodes certain assumptions about everything being classes, and subClassOf being the main linking predicate? For example, would this cause 1D to drop out of some ontology browser displays?

But I think we also need a definition of 1D and some use cases showing how it gets used. This is where an upper ontology can help. While I think BFO can be a little abstruse at times, it does provide a framework for distinguishing things like variables vs projections in space etc. Going along with this, I think 1D is too short a name for a class, because I don't know from the name what it is. Is it literally the value for a unit "dimension" or can it be used for a 1D entity (e.g a conceptual line between Glasgow and Edinburgh)...?